modules/dop/services/issue/issue.go

```diff
@@ -621,6 +640,28 @@ func (svc *Issue) UpdateIssue(req apistructs.IssueUpdateRequest) error {
 		issueStreamFields[field] = []interface{}{canUpdateFields[field], v}
 	}
 
+	if issueModel.Type == apistructs.IssueTypeBug || issueModel.Type == apistructs.IssueTypeTask {
+		if _, ok := changedFields["plan_finished_at"]; ok {
+			if _, err := validator.TryGetIteration(*req.IterationID); err != nil {
+				return apierrors.ErrUpdateIssue.InternalError(err)
+			}
+			if err := validator.validateIteration(req.PlanFinishedAt.Value()); err != nil {
```

Comment:
will `req.PlanFinishedAt.Value()` cause panic?

Reply:
no, req.PlanFinishedAt is type of IssueTime